### DISCUSSION TITLE
Enable GPUParticles3D sub-emitter amount to be independent of main emitter

### DISCUSSION THREAD

**Discussion by Mikeysax** [godotengine/godot-proposals#5824] [Engine Core]
**TLDR:**

- Particle sub-emitters details like amount of particles should be defined on each individual particle emitter
- Particle emitters should retain it's properties and visual representation regardless of if it's used as a sub-emitter or individually.

**Details:**

Currently, in Godot 4, when attaching a sub-emitter, the amount used on the main emitter overrides the sub-emitter. 

This prevents certain VFX from easily being done and which can be done with other particle systems in other engines like Unreal Engine or Unity.

For example, I have this GPUParticles3D that creates a cloud:

![image](https://user-images.githubusercontent.com/15805689/203062090-ddd00b9b-2cba-4ea8-90fa-5b687efcfec2.png)

When attaching this GPUParticles3D as a sub-emitter, I would expect to see multiple instances of the above screenshot. Instead, I see something like this:

![image](https://user-images.githubusercontent.com/15805689/203062363-4a772fd7-9030-4260-96b0-8fbbd810f107.png)

This limitation prevents someone from easily spawning of multiple particle instances in a similar manner and also prevent other VFX from easily being achieved. 

This change would also be more flexible than the current way in which the total particles emitted on collision and at end particles are generated:

![image](https://user-images.githubusercontent.com/15805689/203065298-1f8c86fc-ccaa-41ab-be9a-e01c389643a1.png)

Where the amount spawn would be included on the child instance instead of the parent instance; I believe this is more intuitive for all use cases and also more similar to how other implementations in other engines.
